iiab-support

```diff
@@ -55,13 +55,14 @@ else
     else
         echo "openvpn_install: True" >> /etc/iiab/local_vars.yml
     fi
-    
+
     if [ -d /opt/iiab/iiab ]; then
         cd /opt/iiab/iiab
         #CWD=`pwd`
         #export ANSIBLE_LOG_PATH="$CWD/iiab-install.log"
         export ANSIBLE_LOG_PATH="/opt/iiab/iiab/iiab-install.log"
-        ansible-playbook -i $INVENTORY $PLAYBOOK --connection=local
+        ansible -m setup -i $INVENTORY localhost --connection=local -e ansible_python_interpreter=/usr/bin/python3 | grep python
+        ansible-playbook -i $INVENTORY $PLAYBOOK --connection=local -e ansible_python_interpreter=/usr/bin/python3
```

Comment:
Why are both lines needed when they were not in the past?

(What is the 1st of these 2 lines doing, both here and in general?)

Reply:
see below